src/daemon/abrt-action-save-package-data.c

```diff
@@ -25,15 +26,18 @@
     "/usr/bin/nspluginviewer, /usr/lib*/firefox/plugin-container"
 #define DEFAULT_BLACKLISTED_PKGS "bash, mono-core, nspluginwrapper, strace, valgrind"
 #define DEFAULT_GPG_KEYS_DIR "/etc/pki/rpm-gpg"
-#define DEFAULT_INTERPRETERS "python, python2, python2.7, python3, python3.3, " \
-    "python3.4, python3.5, python3.6, python3.7, perl, perl5.16.2"
+#define DEFAULT_INTERPRETERS_REGEX \
+    "(perl ([:digit:][.][:digit:]+[.][:digit:])?" \
+    "|" \
+    "php" \
+    "|" \
+    "(platform-)? python ([:digit:]([.][:digit:])?)?)$"
```

Comment:
For some reason, the regex doesn't work for me with `[:digit:]` but it does with `\d` (tried on a trivial harness).

Reply:
Oh right, it should be `[[:digit:]]` (with the two pairs brackets).

Reply:
Yeah, I used `\d` at first, but then in an attempt to make the regex more readable I've changed it to `:digit:` :man_facepalming:

Fixed.